rules/windows/builtin/win_net_ntlm_downgrade.yml

```diff
@@ -24,10 +24,10 @@ logsource:
 detection:
     selection1:
         EventID: 13
-        TargetObject: 
-            - '*SYSTEM\\*ControlSet*\Control\Lsa\lmcompatibilitylevel'
-            - '*SYSTEM\\*ControlSet*\Control\Lsa*\NtlmMinClientSec'
-            - '*SYSTEM\\*ControlSet*\Control\Lsa*\RestrictSendingNTLMTraffic'
+        TargetObject|endswith: 
+            - 'SYSTEM\\*ControlSet*\Control\Lsa\lmcompatibilitylevel'
+            - 'SYSTEM\\*ControlSet*\Control\Lsa*\NtlmMinClientSec'
+            - 'SYSTEM\\*ControlSet*\Control\Lsa*\RestrictSendingNTLMTraffic'
```

Comment:
As you are using a value modifier here, it's gonna consider all `*` as it is, not as a mask.
So better to reconstruct it this way:

```
        TargetObject|contains|all: 
            - 'SYSTEM\'
            - 'ControlSet'
            - '\Control\Lsa'
        TargetObject|endswith: 
            - '\lmcompatibilitylevel'
            - '\NtlmMinClientSec'
            - '\RestrictSendingNTLMTraffic'
```

Reply:
Fixed :)